app/src/main/java/com/duckduckgo/app/global/db/AppDatabase.kt

```diff
@@ -552,6 +554,19 @@ class MigrationsProvider(val context: Context) {
         }
     }
 
+    val MIGRATION_42_TO_43: Migration = object : Migration(42, 43) {
+        val fireproofLoginDetector = OldSettingsDataStore()
```

Comment:
Maybe rename the variable `fireproofLoginDetector` and inner class `OldSettingsDataStore` to something more obvious like: `val fireproofSettingsConverter = FireproofSettingsConverter()`

Reply:
Yeah, I agree it will improve readability. Although, I called it `OldSettingsDataStore` to reuse it in case we need to migrate more settings from this class and to be consistency with the other similar class (OldOnboardingStore) in the same file. What do you think? Even if we decide to keep that class name, I would need to change the variable name as it is very confusing, not sure why I called it `fireproofLoginDetector` 🤷‍♀️

Reply:
Okay, I didn't see that a naming convention had been set with `OldOnboardingStore` (Not my favorite but I'm all for consistency 😅).

Just change the variable name then 👍